pkg/apis/certificates/validation/validation_test.go

```diff
@@ -378,7 +378,7 @@ func TestValidateCertificateSigningRequestCreate(t *testing.T) {
 	for name, test := range tests {
 		t.Run(name, func(t *testing.T) {
 			el := ValidateCertificateSigningRequestCreate(&test.csr, test.gv)
-			if !reflect.DeepEqual(el, test.errs) {
```

Comment:
Can you tell why this solves the issue?

also, `cmp.Diff` would probably give you more information for debugging than `reflect.DeepEqual`

Reply:
I'm thinking that the problem is on `errs: field.ErrorList{}`. Probably better to use just leave it at `nil`

Reply:
No, the validation methods return `field.ErrorList{}`, not `nil`.

Reply:
can you replace the check with `cmp.Diff` that will give us information if we see still see the problem in the jobs.